types/core-util-is/core-util-is-tests.ts

```diff
@@ -0,0 +1,76 @@
+import util = require('./');
```

Comment:
This should have been `require("core-util-is")`; I'm not sure why the tooling did not complain about this.